api/login.go

```diff
@@ -25,22 +27,22 @@ func (cli *Client) LoginWithGithub(endpoint string, githubToken string) (*User,
 	return userResp.User, nil
 }
 
-func (cli *Client) LoginWithVault(endpoint string, vaultToken string) (*User, error) {
-	url := endpoint + pathPrefix + "/login"
-	req, err := http.NewRequest("POST", url, nil)
+func (cli *Client) LoginWithVault(vault_url string, githubToken string) (*User, error) {
+	req, err := http.NewRequest("POST",
+		vault_url+"/v1/auth/github/login",
```

Comment:
pathPrefix = "/v1"
So I thought we can use this here as well but I think this is not big deal